docs/developers/contributing/documentation/index.md

```diff
@@ -43,10 +43,33 @@ or [editing an existing file](https://docs.github.com/en/repositories/working-wi
 on the [napari/docs](https://github.com/napari/docs) GitHub repository.
 It's best if you first [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) the [napari/docs](https://github.com/napari/docs) repository to your own GitHub account, create a [feature branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch), upload/create/edit files through the GitHub web interface, and then [open a pull request from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) back to [napari-docs](https://github.com/napari/docs).
 
-You will also be able to [preview the documentation](doc_view_ci) as it would appear on [napari.org](https://napari.org/dev) by
-using the `Check the rendered docs here!` action at the bottom of your PR which will go to a preview site on [CircleCI](https://circleci.com/).
-A member of the maintenance
-team will help with updating the [napari.org](https://napari.org/dev) table of contents where necessary (by placing a reference to your new file in [docs/_toc.yml](update-toc)) and making sure your documentation has built correctly.
+When you submit your PR, a number of jobs will be kicked off, including generation of a preview of 
+the documentation. By default, they will use the `slimfast` build ("make target"), which is fast, because
+it doesn't build any content from outside the `docs` repository and doesn't execute notebook cells.
+This is great for seeing the copy and formatting, but if you want to preview other elements,
+you can trigger more complete builds by commenting on the PR with:
```

Comment:
```suggestion
When you submit your PR, CI will kick off several jobs, including generation of a preview of 
the documentation. By default, CI will use the fast `slimfast` build ("make target"), which
doesn't build any content from outside the `docs` repository and doesn't execute example notebook cells.
The `slimfast` build  is great for seeing the copy and formatting. If you want to preview other elements,
you can trigger more complete builds by commenting on the PR with:
```

Reply:
I worry about confusion with the added "examples": the notebook cells referred to here are not the examples--these are "outside the docs repo"--but rather the cells in the notebooks that make up the docs.
Not sure how to capture this clearly though without being too wordy and needing a TLDR.

Reply:
Maybe replace in my suggestion "and doesn't execute example notebook cells" with "or run notebook cells". Alternatively just strike "doesn't build any content ... notebook cells". The second option may be clearer.

Reply:
Ok, i riffed on that a bit. Lemme know what you think.